frame/proxy/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "pallet-proxy"
+version = "2.0.0-rc2"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "Apache-2.0"
+homepage = "https://substrate.dev"
+repository = "https://github.com/paritytech/substrate/"
+description = "FRAME utilities pallet"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+serde = { version = "1.0.101", optional = true }
+codec = { package = "parity-scale-codec", version = "1.3.0", default-features = false }
+frame-support = { version = "2.0.0-rc2", default-features = false, path = "../support" }
+frame-system = { version = "2.0.0-rc2", default-features = false, path = "../system" }
+sp-core = { version = "2.0.0-rc2", default-features = false, path = "../../primitives/core" }
+sp-runtime = { version = "2.0.0-rc2", default-features = false, path = "../../primitives/runtime" }
+sp-std = { version = "2.0.0-rc2", default-features = false, path = "../../primitives/std" }
+sp-io = { version = "2.0.0-rc2", default-features = false, path = "../../primitives/io" }
```

Comment:
```suggestion
```

Reply:
I tried to do a cleanup a while back and almost all imports of sp_io in frame are unused. It will be stripped out of the  final release build for sure, but still, uncool IMO :D